.github/workflows/workflow-dispatch-job.yml

```diff
@@ -83,7 +83,7 @@ jobs:
           echo "SCCACHE_IDLE_TIMEOUT=32768" >> $GITHUB_ENV
           echo "SCCACHE_S3_USE_SSL=true" >> $GITHUB_ENV
           echo "SCCACHE_S3_NO_CREDENTIALS=false" >> $GITHUB_ENV
-      - name: Run command
+      - name: Run command # Do not change this step's name.
```

Comment:
out of curiosity, where does the dependency on the name come from?

Reply:
I'm currently parsing the total runtime of the job and the runtime of the command step separately to see how much overhead is spent initializing the machine, downloading images, etc.

A quick check of `((job_time - command_time) / command_time)` across the jobs might be interesting.

Reply:
These stats are printed out in the log:

```
job time | command time | runner overhead as much-too-precise floating point percentage | job name
```

It's mildly interesting to have this info broken down by project / job / os / etc, but at the end of the day there's not much we can do with it.

![image](https://github.com/NVIDIA/cccl/assets/58744/7259dc4b-6262-4702-a1f9-68bed9b0e693)